lib/core/plugin/pluginContext.js

```diff
@@ -175,6 +175,34 @@ class PluginContext {
         }
       });
 
+      Reflect.defineProperty(this.accessors, 'subscription', {
+        enumerable: true,
+        value: {
+          register: (connectionId, index, collection, filters) => {
+            const request = new Request(
+              {
+                action: 'subscribe',
+                body: filters,
+                collection,
+                controller: 'realtime',
+                index,
+              },
+              {
+                connectionId: connectionId,
+              });
+            return kuzzle.ask(
+              'core:realtime:subscribe',
+              request
+            );
+          },
+          unregister: (connectionId, roomId, notify) => 
```

Comment:
I would rather use an option object for the last parameter because it's more explicit than just passing a boolean
```suggestion
          unregister: (connectionId, roomId, { notify=true } = {}) => 
```